build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
   }
   dependencies {
     if (JavaVersion.current().isJava7Compatible()) {
-      classpath 'com.netflix.nebula:nebula-publishing-plugin:2.0.0'
+      classpath 'com.netflix.nebula:nebula-publishing-plugin:2.2.+'
```

Comment:
Sorry, that was an experiment, as I tried to use the nebula plugin. Forgot to revert this change.